packages/imperative/src/config/src/Config.ts

```diff
@@ -117,14 +117,6 @@ export class Config {
         secure: ConfigSecure
     };
 
-    // _______________________________________________________________________
-    /**
-     * Constructor for Config class. Don't use this directly. Await `Config.load` instead.
-     * @param opts Options to control how Config class behaves
-     * @private
-     */
-    private constructor(public opts?: IConfigOpts) { }
```

Comment:
I believe we need to keep this constructor, as removing the definition of `public opts` could be a breaking change.

Reply:
I think to continue moving this pr forward, I'll just put it back and we can keep this as an open discussion point for later, possibly during a standup